incorrect/flaky test 'verify should reject expired tokens' with real mysql database

STR

1. Run this command (sorry @seanmonstar, may not be Windows compatible)

```
while NODE_ENV=test DB=mysql ./node_modules/.bin/mocha --ui bdd --reporter spec --grep 'verify should reject expired tokens'; do 
    echo "OK?: $? -- $(date)"; 
    sleep 1; 
done
```
Before very long, you will get a test failure:
```
  1) /v1 /verify should reject expired tokens:

      AssertionError: 200 == 400 from assert.equal(res.statusCode, 400);
      + expected - actual

      -200
      +400

      at test/api.js:1652:16
      at Promise.b (domain.js:183:18)
```

The reason for the failures it seems, is that we are doing this in the test:
```
INSERT INTO tokens (clientId, userId, email, scope, type, expiresAt, token)
  VALUES (X'dcdb5ae7add825d2',
          X'bffe4e612f3ac658c3fdf8c59b2e3423',
          'b2537329@mozilla.com',
           'a',
           'bearer',
           '2015-07-18 22:07:19.658',
           X'79c4571988d60d8ae3194e07c946046111f98b0d8cbc8dd06fcb0cf115ec398f')
```

Mysql will round a TIMESTAMP to the nearest second (for our schema where have default 1 second precision, which is the normal case). So, sometimes `createdAt` - `expiresAt` is 2 seconds, and sometimes 1 second, hence flaky test.

--- a/test/api.js
+++ b/test/api.js
@@ -1639,7 +1639,7 @@ describe('/v1', function() {
       this.slow(2200);
       return newToken({
         ttl: 1
-      }).delay(1000).then(function(res) {
+      }).delay(1500).then(function(res) {
         assert.equal(res.statusCode, 200);
         assert.equal(res.result.expires_in, 1);
         return Server.api.post({

